### DISCUSSION TITLE
Where does this information go?

### DISCUSSION THREAD

**Discussion by Danlaw7777** [jamf/Setup-Manager#27] [Q&A]
does this go into the prestage profile? or somewhere else?
Configure Setup Manager
Add the Setup Manager pkg to the Prestage. Also create a configuration profile for Setup Manager with the workflow to install and configure the software you want to be installed at this stage.

You need to leave least one panel of Setup Assistant enabled. Otherwise Setup Manager might not launch.

Setup Manager profile will require a userEntry field for userID to know which user to assign the Mac to. This will show a field prompting for "User Email." You can of course add other fields to userEntry at this time, though they are not required.

Example:

<key>userID</key>
<dict>
  <key>placeholder</key>
  <string>first.last@example.com</string>
  <key>validation</key>
  <string>\S+\.\S+\@example\.com</string>
  <key>validationMessage</key>
  <string>Email needs to be for example.com!</string>
  <key>validationMessage.de</key>
  <string>Email muss für example.com sein!</string>
</dict>

**Comment by scriptingosx** [comment]
I don’t understand your question? What are you trying to achieve? 

Maybe join the #jamf-setup-manager channel on MacAdmins Slack for more interactive support?
